### ISSUE TITLE
Feature request: Support pasting images from a copy to the diary

### ISSUE THREAD

**Issue by Murmele** [jendrikseipp/rednotebook#504]
### Versions

RedNotebook version (make sure you have the latest version): 2.19

Operating system and version: linux

### Expected behavior
File is copied to a folder related to this diary and in the diary it self a link to it is created

### Actual behavior
Droping files is not supported

### Steps to reproduce the behavior
Think this pull request must be finished before: #469
1) copy image in the file explorer
2) paste it to the diary

**Comment by jendrikseipp** [comment]
I believe this is a duplicate of #163.
